docs/proposals/README.md

```diff
@@ -1,6 +1,3 @@
 # Design proposals
 
 This directory contains accepted design proposals.
```

Comment:
Maybe for a separate PR, but we barely use the proposals folder. We might as well move the proposals to the https://github.com/DataDog/architecture repo

Reply:
Why not, asking the Platform team (co-owning the folder with docs) about it

Reply:
Sure 😄 
@DataDog/documentation @DataDog/agent-platform, any thought about keeping this here or moving it to the https://github.com/DataDog/architecture repo?

Reply:
No issue with moving it from the docs side